api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -540,11 +550,12 @@ public ResponseEntity<Workspace> updateWorkspace(
       throw e;
     }
 
-    final Workspace editedWorkspace = workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace);
+    final Workspace editedWorkspace =
+        workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace, initialCreditsExpirationService);
 
     workspaceAuditor.fireEditAction(
         originalWorkspace, editedWorkspace, dbWorkspace.getWorkspaceId());
-    return ResponseEntity.ok(workspaceMapper.toApiWorkspace(dbWorkspace, fcWorkspace));
+    return ResponseEntity.ok(editedWorkspace);
```

Comment:
This seems like a good idea. I am curious though, do you know why this was being mapped twice?

Reply:
I think that was just an oversight